stacks/environment-aws/main.tf

```diff
@@ -24,7 +24,7 @@ provider "kubernetes" {
 }
 
 provider "helm" {
-  version         = "1.1.0"
+  version         = "1.0.0"
   
```

Comment:
did you mean to roll back this version?

Reply:
was testing some changes, moved the version back to 1.1.0

Reply:
I can approve this once that change is made, everything else LGTM